notebooks/vertex_genai/labs/gemini_function_calling.ipynb

```diff
@@ -882,6 +882,222 @@
     "Function calling is an incredibly versatile tool! "
```

Comment:
<p>"Let's see how we can implement...."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>